spec.html

```diff
@@ -167,7 +167,7 @@ <h1>Iterator.from ( _O_ )</h1>
             1. Let _usingIterator_ be ? GetMethod(_O_, @@iterator).
             1. If _usingIterator_ is not *undefined*,
               1. Let _iteratorRecord_ be ? GetIterator(_O_, ~sync~, _usingIterator_).
-              1. Let _hasInstance_ be ? OrdinaryHasInstance(%Iterator.prototype%, _iteratorRecord_.[[Iterator]]).
+              1. Let _hasInstance_ be ? OrdinaryHasInstance(*Iterator*, _iteratorRecord_.[[Iterator]]).
```

Comment:
I think this should be `%Iterator%`, not `*Iterator*`.

Reply:
Yep, that makes sense. Done.